File: Array/1929. Concatenation of Array/Solution.rs
impl Solution {
    pub fn get_concatenation(nums: Vec<i32>) -> Vec<i32> {
        let mut res=Vec::with_capacity(nums.len()*2);
        res.resize(nums.len()*2,0);
        for i in 0..nums.len(){
            res[i]=nums[i];
            res[i+nums.len()]=nums[i];
        }
        res
    }
}

File: Array/1929. Concatenation of Array/Solution.go
func getConcatenation(nums []int) []int {
	res := make([]int, 0, len(nums)*2)
	for i := 0; i < 2; i++ {
		for j := 0; j < len(nums); j++ {
			res = append(res, nums[j])
		}
	}
	return res
}